src/WordTokenizers.jl

```diff
@@ -15,7 +20,8 @@ export poormans_tokenize, punctuation_space_tokenize,
        set_tokenizer, set_sentence_splitter,
        rev_tokenize, rev_detokenize,
        toktok_tokenize
-
+export Albert_Version1, Albert_Version2, load, tokenizer, sentence_from_tokens, ids_from_tokens
+export pretrained
```

Comment:
Does this need to be exported? Seems like a global value?

Reply:
`pretrained` will act as a list vocab function
 Albert_Version1 and Alber_version2 will help in downloading using Datadeps 
`load(ty::Type{T}, name::String)
`I think we need to export them in order to remain DataType. It's something similar to embedding.jl 
would you like me to change it?

Reply:
@aviks